src/Symfony/Component/Process/PhpExecutableFinder.php

```diff
@@ -62,6 +62,10 @@ public function find($includeArgs = true)
             }
         }
 
+        if (is_executable($php = PHP_BINDIR.'/php'.('\\' === DIRECTORY_SEPARATOR ? '.exe' : ''))) {
```

Comment:
This logic does not take into account the `PATHEXT` env var as done in `ExecutableFinder`.

Reply:
php binaries always are `.exe`, in php_bindir